apps/web/app/features/stacking/start-pooled-stacking/components/choose-pooling-duration.tsx

```diff
@@ -9,8 +9,8 @@ export function ChoosePoolingDuration() {
         <Stack gap="space.01">
           <styled.p textStyle="label.03">Indefinite</styled.p>
           <styled.p textStyle="caption.01">
-            The pool stacks your STX for up to 12 cycles. You can revoke anytime, but they stay
-            locked until the cycle ends. Revoke before restacking to regain access.
+            The pool will commit your STX for Stacking for up to 12 cycles (with about two weeks per cycle). You can revoke anytime, but they stay
```

Comment:
passive voice

Reply:
Because it's in future tense, suggesting this change?

"The pool commits your STX to Stacking for up to 12 cycles. You can revoke before a cycle restarts to regain access."